test/test_config/hostfactory_policy.yml

```diff
@@ -0,0 +1,11 @@
+- !policy
+  id: hostfactory_policy
+
+  body:
+    - !layer some_vms
```

Comment:
can we use `!group` instead? i think that `layer` will be deprecated sometime in the future

Reply:
can `host-factory` work with `group`?

Reply:
yes, just checked

Reply:
checked with group and get `422 Client Error: Unprocessable Entity for url: https://localhost/policies/cucumber/policy/root. The policy was empty or its contents were not in valid YAML.`

Reply:
that's odd. can you share the policy you tried to load?

Reply:
```
- !policy
  id: hostfactory_policy

  body:
    - !group some_vms

    - !host-factory
      id: some_host_factory
      annotations:
        description: Factory to create new vms
      groups: [ !group some_vms ]

```